resources/compass/charts/connectivity-adapter/templates/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1beta2
```

Comment:
```suggestion
apiVersion: apps/v1
```
`apps/v1beta2` is deprecated and is not supported from Kubernetes 1.16 up.

Reply:
thanks @Szymongib, I will solve this in another PR